L1Trigger/L1TMuonOverlapPhase1/src/Omtf/GhostBuster.cc

```diff
@@ -32,6 +32,9 @@ AlgoMuons GhostBuster::select(AlgoMuons refHitCands, int charge) {
     }
     if ((*it1)->getQ() > 0 && !isGhost)
       refHitCleanCands.emplace_back(new AlgoMuon(**it1));
+
+    if (refHitCleanCands.size() >= 3)
+      break;

```

Comment:
If only for my edification, where is the `3` coming from?

Reply:
This is a backport of #39624, so @kbunkow or another muon trigger expert should answer.